acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/KeyLoadAndSignAcceptanceTest.java

```diff
@@ -128,6 +133,34 @@ public void unusedFieldsInRequestDoesNotAffectSigning() throws Exception {
     assertThat(response.getStatusCode()).isEqualTo(HttpResponseStatus.OK.code());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"/signer/block"}) // , "/signer/attestation"})
+  public void ableToSignUsingHashicorp(final String artifactSigningEndpoint)
+      throws ExecutionException, InterruptedException {
+    final String configFilename = keyPair.publicKey().toString().substring(2);
+    final DockerClientFactory dockerClientFactory = new DockerClientFactory();
+    final HashicorpNode hashicorpNode =
+        HashicorpNode.createAndStartHashicorp(dockerClientFactory.create(), true);
```

Comment:
Do we need to shutdown hashicorp node or that happening automatically somewhere?

Reply:
done

Reply:
can't see where the shutdown is being called